settings.gradle

```diff
@@ -80,3 +85,11 @@ buildCache {
         enabled = !isCiServer
     }
 }
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
```

Comment:
Main thing here is whether this is OK as far as our IP lawyers are concerned!

Reply:
I've reached out and asked.